src/js/pensions/config/form.js

```diff
@@ -176,6 +178,61 @@ const formConfig = {
               nationalGuard: _.set('properties.address', address.schema(fullSchemaPensions), nationalGuard)
             }
           }
+        },
+        powAndSeverance: {
+          path: 'military/pow-severance',
+          title: 'POW Status & Severance Pay',
+          uiSchema: {
+            'ui:title': 'POW Status & Severance Pay',
+            'ui:order': ['view:powStatus', 'powStatus', 'powDateRange', 'view:severancePay', 'severancePay', 'separation', 'severance', 'retirement'],
+            powStatus: {
+              'ui:title': 'Have you ever been a POW?',
+              'ui:widget': 'yesNo'
+            },
+            powDateRange: _.set('ui:options.expandUnder', 'powStatus', dateRangeUI(
+                  'Start of confinement',
+                  'End of confinement',
+                  'Confinement start date must be before end date'
+            )),
+            severancePay: {
+              'ui:title': 'Have you received any type of separation/severance retired pay?',
+              'ui:widget': 'yesNo'
+            },
+            separation: {
```

Comment:
I think you need titles for the amount and type fields in each of these objects (though you can wait until we hear back from Alex to add them, if that's easier).